src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/BDDAlgebra.cs

```diff
@@ -26,6 +15,15 @@ internal enum BoolOp
     /// </summary>
     internal abstract class BDDAlgebra : IBooleanAlgebra<BDD>
```

Comment:
I know you are not renaming any of these here, but we might consider renaming all of the `BDD` -> `BinaryDecisionDiagram`. Granted for some of these types it would make it a very long type name (like for the RangeConverter) but IMO it might make the code more readable.

Reply:
Yup, there's still a lot of renaming left to do :)